app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: :User, foreign_key: :user_id
+  belongs_to :post, class_name: :Post, foreign_key: :post_id
+
+  private
+
+  def update_post_count
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] @simplegoose Kindly consider setting up the `update_like_count` correctly as you did for the `update_comment_count` method.